gtsfm/scene_optimizer.py

```diff
@@ -177,10 +177,11 @@ def create_computation_graph(
 
         # ensure cycle consistency in triplets
         cycle_consistent_graph = dask.delayed(cycle_consistency.filter_to_cycle_consistent_edges)(
-            i2Ri1_graph_dict, i2Ui1_graph_dict, two_view_reports_dict
+            i2Ri1_graph_dict, i2Ui1_graph_dict, v_corr_idxs_graph_dict, two_view_reports_dict
         )
         i2Ri1_graph_dict = cycle_consistent_graph[0]
         i2Ui1_graph_dict = cycle_consistent_graph[1]
+        v_corr_idxs_graph_dict = cycle_consistent_graph[2]
 
```

Comment:
curious why we have cycle consistency check in scene optimizer? Seems like it should belong in MVO?

Reply:
could go in either, i don't have too strong of feelings about where it goes

Reply:
Created an issue here: https://github.com/borglab/gtsfm/issues/289